files/en-us/tools/tools_toolbox/index.html

```diff
@@ -56,22 +56,22 @@ <h3 id="Toolbox-hosted_tools">Toolbox-hosted tools</h3>
 
 <h3 id="Extra_tools">Extra tools</h3>
 
-<p>Next there's an array of buttons that can be added or removed in the <a href="#settings">developer tool settings</a>. By default this array includes:</p>
+<p>Next there's an array of buttons that can be added or removed in the <a href="/en-US/docs/Tools/Settings">developer tool settings</a>. By default this array includes:</p>
 
 <ul>
  <li><a href="/en-US/docs/Tools/Web_Console#the_split_console">Toggle split console</a></li>
  <li><a href="/en-US/docs/Tools/Responsive_Design_Mode">Responsive Design Mode</a></li>
  <li><a href="/en-US/docs/Tools/Working_with_iframes">Select a frame as the currently targeted document</a> (this is only included by default from Firefox 41 onwards).</li>
 </ul>
 
-<p>The following tools are not included in the toolbar by default, but you can add them in the <a href="#settings">settings</a>:</p>
+<p>The following tools are not included in the toolbar by default, but you can add them in the <a href="#settings_2">settings</a>:</p>
```

Comment:
```suggestion
<p>The following tools are not included in the toolbar by default, but you can add them in the <a href="/en-US/docs/Tools/Settings">settings</a>:</p>
```

You might as well redirect to the same page as you did above.